Sources/TensorFlow/Operators/Math.swift

```diff
@@ -424,6 +424,20 @@ internal func _vjpLog<T: TensorFlowFloatingPoint>(
     return (log(x), { v in v / x })
 }
 
+/// Computes the logarithm of `1 + x` element-wise.
+@inlinable
+@differentiable(vjp: _vjpLog1p)
+public func log1p<T: TensorFlowFloatingPoint>(_ x: Tensor<T>) -> Tensor<T> {
+    return Raw.log1p(x)
```

Comment:
FYI, Swift supports implicit returns for single line functions now so you can just write `Raw.log1p(x)`.

Reply:
That's cool to know thanks! I'll use that going forward.